src/views/LocationEditors/LocationFromAddress.vue

```diff
@@ -11,12 +18,128 @@
       location: null,
       valid: Boolean,
     },
-    methods: {
-      locationChanged(value) {
-        //this.$emit('update:valid', ...);
-        //this.$emit('update:location', value);
+    async created() {
+        let geoData = await this.getGrouppedData()
```

Comment:
This should be try / catched with a proper error handling if the data could not be loaded

Reply:
We already had the case that the format of geocoding may get updated. In that case, people still having an older version in cache should be warned with a proper message and asked to reload the page if the problem still occurs.